src/lib/operations/create-user/createUser.ts

```diff
@@ -0,0 +1,46 @@
+import { IdTokenWithData, updateAuth0User } from "../../utils/auth/auth0";
+import { CreateUserIdTokenData } from "./create-user.handler";
+import { CONFIG } from "../../server/config/config.utils";
+import { delay } from "../../utils/tools/delay";
+import { createKeyRing, createSignKey, createEncryptDecryptKey, createImportJob } from "./createUser.utils";
+import { notifyUserCreationOnSlack } from "../../utils/slack/slack.utils";
+
+export async function createUser(
+  idToken: IdTokenWithData<CreateUserIdTokenData>,
+  importOnly: boolean,
+) {
+  console.log(`createUser(${ importOnly ? "importOnly" : "generateKeys" })`);
+
+  // TODO: Check if the keyRing / signKey / encryptDecryptKey already exists but the keys were never generated or
```

Comment:
This might become a concern if a user is created but no keys are generated and imported, and then the user comes back and tries to sign up again.